src/Symfony/Component/Serializer/Mapping/Factory/CacheClassMetadataFactory.php

```diff
@@ -32,6 +32,8 @@ class CacheClassMetadataFactory implements ClassMetadataFactoryInterface
      */
     private $cacheItemPool;
 
+    private $loadedClasses = [];
```

Comment:
The name is misleading. It's actually an array of `ClassMetadata`.

Reply:
I followed the name of the delegated class https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Mapping/Factory/ClassMetadataFactory.php#L31